components/button/package.json

```diff
@@ -32,7 +32,8 @@
 		"@westpac/core": "^1.0.0",
 		"@westpac/hooks": "^1.0.0",
 		"@westpac/icon": "^1.0.0",
-		"prop-types": "^15.7.2"
+		"prop-types": "^15.7.2",
+		"react-router-dom": "^5.1.2"
```

Comment:
That can't be right!

Reply:
This has to go into the devDependencies or we ship this with the component

Reply:
Odd, I ran the flag to add it to devDependencies but it seems like it didn't work...